packit/exceptions.py

```diff
@@ -38,6 +38,7 @@ def __init__(
         stderr_output: Union[str, bytes] = None
     ):
         super().__init__(*args)
+        # we can't use utils.ensure_str here b/c utils imports from exceptions :<
```

Comment:
What about some other place so we can use the function..;)

Reply:
oki, let's merge this and I move the func in the next PR cuz' I want to have this off my chest :D

Reply:
never mind, some tests failed; will fix those and move the func